### PR TITLE
Finish linter

### PR DESCRIPTION
**Pull request by obnoxiousorc**

### CODE REVIEW

**Review by jonjensen — CHANGES_REQUESTED** [review]
I don't see a way to comment on lines not visible in the diff, but here are a couple of others:

```
bool(re.match(r'.*\t.*', l.line)
```

Since you are looking anywhere in the string, not matching at beginning and end, use `re.search()` instead and you can get rid of the `.*` at the beginning and end.

But also since you're looking for a single character, not a regex, you can use faster regular substring search:

```
l.line.find('\t') >= 0
```

which may be a microoptimization, but oh well.

This appears twice:

```
without_code = re.sub(r'`[^`]*`', '', line.line)
```

seems like it should be ```r'[^`]+'``` that is, always requiring something in the code block, not an empty one, right?

Finally, do you have in mind a clear distinction between warnings and errors? Just curious. I bet on rare occasion we may need to use 3-backtick quoting on a single line when we're quoting a backtick, even though normally we wouldn't want that. And just wondered about how to decide.

File: bin/blog-lint.py [hunk 1]
```diff
@@ -264,18 +350,12 @@ def check_spellings(line):
             'forKeepers': True,
         },
         {
-            'regex': r'[^\s]/[^\s]',
+            'regex': r'[^\s]/[^​]',
```
[the remark this answers is not in the document]
Comment (obnoxiousorc) [reply bin/blog-lint.py:353]: I'm fine with either, and I did also fix the bit you mentioned about `without_code`. Let me know which one you prefer here and I'll push my changes.

[the remark this answers is not in the document]
Comment (jonjensen) [reply bin/blog-lint.py:353]: Let's go with explicit `\u200b` then. I think many people will totally miss it otherwise.

**Comment by jonjensen** [comment]
@obnoxiousorc You're still going to push a new commit to wrap this up, right?

**Comment by obnoxiousorc** [comment]
@jonjensen Sorry, slipped my mind this week. Pushed now though, looks like the `re` module supports `\u200b` in an `r'...'` string just fine!

**Pull request merged**